src/new/wallet.js

```diff
@@ -533,7 +715,96 @@ class HathorWallet extends EventEmitter {
     };
   }
 
+  /**
+   * Get balance for a token (same as old method to be used for compatibility)
+   *
+   * @params {string} tokenUid Token uid
+   *
+   * @return {Object} Object with balance { available, locked }
+   *
+   * @memberof HathorWallet
+   * @inner
+   **/
+  getOldBalance(tokenUid) {
```

Comment:
Should we mark it as deprecated? I still didn't understand what's changing in the storage so we need this compatibility methods.

Reply:
I'd say `_getBalance()` or `_getRawBalance()`. What do you think?

Reply:
Changed to `_getBalanceRaw`